module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaApplicationKeyCredential.md

```diff
@@ -8,34 +18,50 @@ schema: 2.0.0
 # Remove-EntraBetaApplicationKeyCredential
 
 ## Synopsis
+
 Removes a key credential from an application.
 
 ## Syntax
 
-```
-Remove-EntraBetaApplicationKeyCredential -ObjectId <String> -KeyId <String>[<CommonParameters>]
+```powershell
+Remove-EntraBetaApplicationKeyCredential
+ -ObjectId <String> 
+ -KeyId <String> 
+ [<CommonParameters>]
 ```
 
 ## Description
-The Remove-EntraBetaApplicationKeyCredential cmdlet removes a key credential from an application.
+
+The `Remove-EntraBetaApplicationKeyCredential` cmdlet removes a key credential from an application.
+
+An application can use this command along with `New-EntraBetaApplicationKeyCredential` to automate the rolling of its expiring keys.
 
 ## Examples
 
 ### Example 1: Remove a key credential
-```
-PS C:\> Remove-EntraBetaApplicationKeyCredential -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -KeyId "6aa971c6-3040-45df-87ed-581c8c09ff2b"
+
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+
+$params = @{
+    ObjectId = '33334444-dddd-5555-eeee-6666ffff7777'
+    KeyId = 'aaaaaaaa-0b0b-1c1c-2d2d-333333333333'
+}
+
+Remove-EntraBetaApplicationKeyCredential @params
```

Comment:
In example description we need t to add used parameters description for ObjectId and KeyId